__tests__/app.test.js

```diff
@@ -35,4 +35,25 @@ describe("/api/topics", () => {
             expect(message).toBe("Path not found")
         })
     })
+})
+
+describe("/api", () => {
+    test("GET 200: Should return an object", () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({body}) => {
+            expect(typeof body).toBe("object")
+        })
+    })
+    test("GET 200: Should return an object with current endpoints with all information matching the endpoints.json file", () => {
+        const endpoints = require(`${__dirname}/../endpoints.json`)
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.length).toBe(endpoints.length)
```

Comment:
is endpoints an array? length check not necessary here, toEqual assertion is enough :D